.prettierrc.json

```diff
@@ -4,5 +4,6 @@
 	"singleQuote": true,
 	"trailingComma": "all",
 	"useTabs": true,
-	"tabWidth": 2
+	"tabWidth": 2,
+	"plugins": ["prettier-plugin-organize-imports"]
```

Comment:
This shouldn't be necessary, as the VSCode extension picks plugins up automatically

Reply:
Oh so `plugins` can be removed?